Walnut/src/Walnut/Input/Input.h

```diff
@@ -2,19 +2,28 @@
 
 #include "KeyCodes.h"
 
+#include <functional>
+#include <unordered_map>
 #include <glm/glm.hpp>
 
 namespace Walnut {
 
 	class Input
 	{
 	public:
+		static void InitKeysCallBack();
+
+		static void SetKeyCallback(KeyCode keycode, std::function<void()> func);
```

Comment:
I would suggest to pass the `KeyCode` to the callback, so that one function _could_ be used for multiple keys.